Sources/Widgets/Widgets3D/ResliceCursorWidget/state.js

```diff
@@ -0,0 +1,118 @@
+  .createBuilder()
+  .addField({ name: 'point1', initialValue: [-factor, 0, 0] })
+  .addField({ name: 'point2', initialValue: [factor, 0, 0] })
+  .addField({ name: 'rotationPoint1', initialValue: [-rotationFactor, 0, 0] })
+  .addField({ name: 'rotationPoint2', initialValue: [rotationFactor, 0, 0] })
+  .addField({ name: 'color', initialValue: axisYColor })
+  .addField({ name: 'name', initialValue: 'axisYinZ' })
+  .addField({ name: 'planeName', initialValue: 'Coronal' })
+  .build();
+
+const axisZinX = vtkStateBuilder
+  .createBuilder()
+  .addField({ name: 'point1', initialValue: [0, -factor, 0] })
+  .addField({ name: 'point2', initialValue: [0, factor, 0] })
+  .addField({ name: 'rotationPoint1', initialValue: [0, -rotationFactor, 0] })
+  .addField({ name: 'rotationPoint2', initialValue: [0, rotationFactor, 0] })
+  .addField({ name: 'color', initialValue: axisZColor })
+  .addField({ name: 'name', initialValue: 'axisZinX' })
+  .addField({ name: 'planeName', initialValue: 'Axial' })
+  .build();
+const axisZinY = vtkStateBuilder
+  .createBuilder()
+  .addField({ name: 'point1', initialValue: [-factor, 0, 0] })
+  .addField({ name: 'point2', initialValue: [factor, 0, 0] })
+  .addField({ name: 'rotationPoint1', initialValue: [-rotationFactor, 0, 0] })
+  .addField({ name: 'rotationPoint2', initialValue: [rotationFactor, 0, 0] })
+  .addField({ name: 'color', initialValue: axisZColor })
+  .addField({ name: 'name', initialValue: 'axisZinY' })
+  .addField({ name: 'planeName', initialValue: 'Axial' })
+  .build();
+
+export default function generateState() {
+  return vtkStateBuilder
+    .createBuilder()
+    .addStateFromInstance({
+      labels: ['axisXinY'],
+      name: 'axisXinY',
+      instance: axisXinY,
+    })
+    .addStateFromInstance({
+      labels: ['axisXinZ'],
+      name: 'axisXinZ',
+      instance: axisXinZ,
+    })
+    .addStateFromInstance({
+      labels: ['axisYinX'],
+      name: 'axisYinX',
+      instance: axisYinX,
+    })
+    .addStateFromInstance({
+      labels: ['axisYinZ'],
+      name: 'axisYinZ',
+      instance: axisYinZ,
+    })
+    .addStateFromInstance({
+      labels: ['axisZinX'],
+      name: 'axisZinX',
+      instance: axisZinX,
+    })
+    .addStateFromInstance({
+      labels: ['axisZinY'],
+      name: 'axisZinY',
+      instance: axisZinY,
+    })
+    .addField({ name: 'center', initialValue: [0, 0, 0] })
+    .addField({ name: 'activeLine', initialValue: null })
+    .addField({ name: 'activeHandle', initialValue: null })
+    .addField({ name: 'image', initialValue: null })
+    .addField({ name: 'activeViewName', initialValue: '' })
+    .addField({ name: 'sphereRadius', initialValue: 5 })
+    .addField({
+      name: 'updateMethodName',
+    })
+    .addField({ name: 'sagittalPlaneNormal', initialValue: [1, 0, 0] })
+    .addField({ name: 'coronalPlaneNormal', initialValue: [0, -1, 0] })
+    .addField({ name: 'axialPlaneNormal', initialValue: [0, 0, 1] })
```

Comment:
sometimes you call it axial, and sometimes Z, it would be less confusing if you pick only 1 naming convention